.github/workflows/build_and_test.yml

```diff
@@ -244,6 +244,8 @@ jobs:
           pyspark-coursier-
     - name: List Python packages (Python 3.9)
       run: |
+        # TODO(SPARK-36345): Install mlflow>=1.0 and sklearn in Python 3.9 of the base image
+        python3.9 -m pip install 'mlflow>=1.0' sklearn
```

Comment:
sorry actually let's move this right before `export PATH=$PATH:$HOME/miniconda/bin` at `Run tests`.

Reply:
Sure, just moved!